spec/models/like_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  # pending "add some examples to (or delete) #{__FILE__}"
+  subject { User.new(Name: 'Tom', Image: 'https://unsplash.com/photos/F_-0BxGuVvo', Bio: 'Teacher from Mexico.') }
+
+  before { subject.save }
+
+  before(:each) do
+    @post = Post.create(AuthorId: subject.id, Title: 'check post',
+                        Text: 'Unit test for the method of most_recent_posts')
+    @comment = Comment.create(AuthorId: subject.id, PostId: @post.id,
+                              Text: 'Unit test for the method of most_recent_comments')
+  end
+
+  it 'Correct AuthorId is present' do
+    like = Like.new(AuthorId: subject.id, PostId: @post.id)
+
+    id = like.AuthorId
+
+    expect(id).to eq(subject.id)
+  end
```

Comment:
- [ ] As the previous reviewer mentioned kindly make sure to fix these errors so the rspec tests are passing 👇🏼 
![Screenshot_20230128_204641](https://user-images.githubusercontent.com/25227022/215287944-388dc438-b606-40e3-8ca3-3875f7f5b1a1.png)